acra/guides/trying-acra-with-docker/_index.md

```diff
@@ -0,0 +1,403 @@
+|----------|------------------------------|-----------------------------------------|
+| 9191/tcp | AcraServer/AcraConnector API | zonemode                                |
+| 9393/tcp | AcraServer                   | absent AcraConnector                    |
+| 9494/tcp | AcraConnector                | present AcraConnector                   |
+| 8000/tcp | AcraWebconfig                | present AcraConnector and SecureSession |
+| 5432/tcp | PostgreSQL                   | PostgreSQL                              |
+| 3306/tcp | MySQL                        | MySQL                                   |
+
+
+### Normal launch
+
+#### Description of components
+
+There are several different dockerfiles in `docker` subdirectory:
+
+* `acra-authmanager.dockerfile` – resulting image with AcraAuthmanager tool
+* `acra-connector.dockerfile` – resulting image with AcraConnector
+* `acra-server.dockerfile` – resulting image with AcraServer
+* `acra-keymaker.dockerfile` – resulting image with AcraKeymaker tool
+* `acra-webconfig.dockerfile` – resulting image with AcraWebconfig component
+* `acra-build.dockerfile` – intermediate image for compiling all the Acra components
+* `mysql-nossl.dockerfile` – MySQL server container with disabled SSL
+* `mysql-ssl.dockerfile` – MySQL server container with example SSL certificates (located at ssl/mysql directory)
+* `postgresql-ssl.dockerfile` – Postgresql server container with example SSL certificates (located at ssl/postgresql directory)
+
+All the images are already built and uploaded to the [Docker Hub Cossack Labs repository](https://hub.docker.com/u/cossacklabs/), so you can use them in a traditional manner:
+
+```bash
+docker run <options> cossacklabs/<component> <arguments>
+```    
+
+You do not need to do it, but you may want to build all the images from current sources manually. To do that, type:
+
+```bash
+make docker
+```    
+
+After that step, you'll get the described above pre-built images (intermediate `acra-build` image will be wiped out after the build is complete).
+
+All docker images have tags. When we build them, we set appropriate tags for each image:
+
+* sliding tags:
+    - `stable` and `latest` - stable branch, recommended, default;
+    - `master` and `current` - master branch of GitHub repository.
+
+* fixed tags:
+    - `<full_commit_tag>` - specify the exact commit in the repository;
+    - `<version>` - choose version tag.
+
+An image usually has two (commit, version) or four (+ branch, latest/current) tags.
+
+#### Compose files
+
+We wanted you to be able to easily try the most useful schemes that we prepared as Docker Compose files in the `docker` subdirectory. The name of each Docker Compose file describes its components and their interconnections in a simple form. For example: `docker-compose.pgsql-nossl-server-ssession-proxy.yml` is a scheme with Postgresql DB, AcraServer, and AcraConnector, connected to AcraServer through the Secure Session link.
```

Comment:
> We wanted you to be able to easily try [...]

And we still want. This is not an apology letter, present tense is better, IMO.

Reply:
+1